document/core/util/mathdefbs.py

```diff
@@ -0,0 +1,121 @@
+from sphinx.ext.mathbase import math
```

Comment:
Can you insert some comment here explaining how this differs from mathdefs? Do you think it's possible to avoid the code duplication with some config parameter?

Reply:
Spent an hour trying to find such a way just now to no avail.
Added comment and a TODO.